packages/apps/public/locales/zh/apps.json

```diff
@@ -28,5 +28,6 @@
   "Waiting for authorization from the extension. Please open the installed extension and approve or reject access.": "扩展等待认证. 打开已经安装的扩展同意或者拒绝他.",
   "You are not connected to a node. Ensure that your node is running and that the Websocket endpoint is reachable.": "没有连接任何节点. 确保你的节点正在运行并且保证websocket终端可访问.",
   "With the Firefox browser connecting to insecure WebSockets ({{wsUrl}}) will fail due to the browser not allowing localhost access from a secure site.": "用Firefox连接不安全的 WebSockets ({{wsUrl}}) 由于浏览器不允许从一个安全的站点访问本地服务, 可能访问失败.",
-  "You are connecting from a secure location to an insecure WebSocket ({{wsUrl}}). Due to browser mixed-content security policies this connection type is not allowed. Change the RPC service to a secure 'wss' endpoint.": "你正在从一个安全的连接访问不安全的 WebSocket ({{wsUrl}}). 由于浏览器最大安全策略这个连接将不被允许访问. 更改RPC服务到一个安全的 'wss' 终端."
+  "You are connecting from a secure location to an insecure WebSocket ({{wsUrl}}). Due to browser mixed-content security policies this connection type is not allowed. Change the RPC service to a secure 'wss' endpoint.": "你正在从一个安全的连接访问不安全的 WebSocket ({{wsUrl}}). 由于浏览器最大安全策略这个连接将不被允许访问. 更改RPC服务到一个安全的 'wss' 终端.",
```

Comment:
Thanks a million for updating these as well - sadly we are a bit behind on the cn translations as it stands (and things have changed a lot with runtime support, so it was a bit of a losing battle)